utility/WiFiStream.h

```diff
@@ -59,19 +60,37 @@ class WiFiStream : public Stream
 public:
   WiFiStream() {};
 
+#ifndef ESP8266
```

Comment:
Right, must be the other way round. The same applies to all the other ESP8266 "excludes" in this file.